src/main/resources/citizen/templates/document/defendantResponseReceipt.html

```diff
@@ -352,6 +352,341 @@
+<div class="section">
+  <div class="section-heading">
+    <strong>The defendant’s financial details</strong>
+  </div>
+  <div class="section-body">
+    <table>
+      <tr class="no-border">
+        <td><strong>Where are they living?</strong></td>
+        <td>
+          {{ residenceTypeDescription }}
+        </td>
+      </tr>
+      <tr>
+        <td colspan="2" class="keep-white-space"><strong>Children:</strong></td>
+      </tr>
+      <tr class="no-border">
+        <td><strong>Do any children live with them?</strong></td>
+        <td>
+        {% if children is defined and children is not empty %}
+          Yes
+        {% else %}
+          No
+        {% endif %}
+        </td>
+      </tr>
+      {% for child in children %}
+        <tr class="no-border">
+          <td>
+            {{ child.ageGroupType }}
+          </td>
+          <td>
+            {{ child.numberOfChildren }}
+          </td>
+        </tr>
+        {% if child.ageGroupType equals 'BETWEEN_16_AND_19' %}
+        <tr>
+          <td><strong>Number of children aged 16 to 19 full-time education or training:</strong></td>
+          <td>
+            {{ child.numberOfChildrenLivingWithYou }}
+          </td>
+        </tr>
+        {% endif %}
+      {% endfor %}
+
+      {% if maintainedChildren is defined %}
+      <tr>
+        <td colspan="2" class="keep-white-space"><strong>Maintenance:</strong></td>
+      </tr>
+      <tr class="no-border">
+        <td><strong>Number of children:</strong></td>
+        <td>
+          {{ maintainedChildren }}
+        </td>
+      </tr>
+      {% endif %}
+      <tr>
+        <td colspan="2" class="keep-white-space"><strong>Financial support:</strong></td>
+      </tr>
+      <tr class="no-border">
+        <td><strong>Do they support anyone else financially?</strong></td>
+        <td>
+        {% if otherDependants is defined %}
+          Yes
+        {% else %}
+          No
+        {% endif %}
+        </td>
+      </tr>
+      {% if otherDependants is defined %}
+      <tr class="no-border">
+        <td><strong>Number of people:</strong></td>
+        <td>
+          {{ otherDependants.numberOfPeople }}
+        </td>
+      </tr>
+      <tr class="no-border">
+        <td><strong>Give details:</strong></td>
+        <td>
+          {{ otherDependants.details }}
+        </td>
+      </tr>
+      {% endif %}
+      <tr>
+        <td colspan="2" class="keep-white-space"><strong>Employment details:</strong></td>
+      </tr>
+      <tr class="no-border">
+        <td><strong>Are they currently working?</strong></td>
+        <td>
+        {% if employment is defined and employment is not empty %}
+          Yes
+        {% else %}
+          No
+        {% endif %}
+        </td>
+      </tr>
+      <tr class="no-border">
+        <td><strong>Type of job:</strong></td>
+        {% if jobType is defined %}
+        <td>
+          {{ jobType }}
+        </td>
+        {% endif %}
+      </tr>
+      <tr>
```

Comment:
This should conditional. We want to display `Who employs them` only when `employment.employers` is defined otherwise there will be header with no data.